upgrade/I.1.3.0/J.1.0.0/roles/install-server/tasks/main.yml

```diff
@@ -51,3 +51,9 @@
     - redis
   tags: install_server
   when: ansible_distribution == 'RedHat'
+
+# Ensure chroot is labelized
+- name: SELinux labelized chroot
+  shell: setfiles -r /var/lib/debootstrap/install/RH7.0-J.1.0.0/openstack-full/ /etc/selinux/targeted/contexts/files/file_contexts /var/lib/debootstrap/install/RH7.0-J.1.0.0/openstack-full/
```

Comment:
I would rather use {{ version }} instead of J.1.0.0, so we can re-use the playbook easily later.